README.md

```diff
@@ -80,100 +71,95 @@ Now we'll convert it using HTML5 Semantic Elements
+### Header
 
 `<header>...</header>`
 
 The HTML `<header>` element represents a group of introductory or navigational aids. It may contain some heading elements but also other elements like a logo, wrapped section's header, a search form, and so on.
 
-#### Main
+### Main
 
 `<main>...</main>`
 
 The HTML `<main>` element represents the main content of  the `<body>` of a document or application. The main content area consists of content that is directly related to, or expands upon the central topic of a document or the central functionality of an application. This content should be unique to the document, excluding any content that is repeated across a set of documents such as sidebars, navigation links, copyright information, site logos, and search forms (unless the document's main function is as a search form).
 
-#### Mark
+### Mark
 
 `<mark>...</mark>`
 
 The HTML Mark Element (`<mark>`) represents highlighted text, i.e., a run of text marked for reference purpose, due to its relevance in a particular context. For example it can be used in a page showing search results to highlight every instance of the searched-for word.
 
-#### Nav
+### Nav
 
 `<nav>...</nav>`
 
 The HTML `<nav>` element (HTML Navigation Element) represents a section of a page that links to other pages or to parts within the page: a section with navigation links.
 
-#### Section
+### Section
 
 `<section>...</section>`
 
 The HTML `<section>` element represents a generic section of a document, i.e., a thematic grouping of content, typically with a heading. Each `<section>`should be identified, typically by including a heading (`<h1>-<h6>` element) as a child of the `<section>` element.
 
-#### Time
+### Time
 
 `<time>`
 
 The HTML `<time>` element represents either a time on a 24-hour clock or a precise date in the Gregorian calendar (with optional time and timezone information).
 
 This element is intended to be used presenting dates and times in a machine readable format. This can be helpful for user agents to offer any event scheduling for user's calendar.
 
-### Older Browser Support
-
-With all new additions to HTML and JavaScript, legacy support for older browsers is a high priority when it comes to user experience. Designing websites to be fully functional for every user typically means building ways to 'gracefully degrade' the syntax and code used in order to support older browsers.  Look at your website in the newest version of Firefox and you will see all the bells and whistles, but even if you're using IE 4 (which you really really shouldn't be), you'll still see something that closely resembles your site.  Previously, this issue applied to semantic elements - since they are a newer way to write HTML, users with old browsers were not be able to interpret the HTML elements.  As of 2018, however, support for semantic elements is now at nearly 98% of all browsers in use, so feel free to use semantic elements in your HTML! 
-
-## Summary
+## Conclusion
```

Comment:
I absolutely agree with removing the cross-compatibility stuff - it's not necessary anymore.